app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
```

Comment:
Instead of using for i in range(len(friends)), it's clearer to iterate directly over the friends list using for friend in friends.

In cafe.visit_cafe(friends[i]), it's recommended to use friend instead of accessing friends[i] directly for clarity and consistency with the iteration.